marbl/handler.go

```diff
@@ -95,12 +100,22 @@ func (h *Handler) unsubscribe(id string) {
 	h.mu.Lock()
 	defer h.mu.Unlock()
 
-	delete(h.subs, id)
+	if framec, ok := h.subs[id]; ok {
+		close(framec)
+		delete(h.subs, id)
+	}
 }
 
 func (h *Handler) subscribe(id string, framec chan<- []byte) {
 	h.mu.Lock()
 	defer h.mu.Unlock()
 
+	if framec, ok := h.subs[id]; ok {
```

Comment:
```suggestion
	if fc, ok := h.subs[id]; ok {
```
avoid confusion with `framec` that gets passed in